src/HydroPowerModels.jl

```diff
@@ -1,35 +1,54 @@
 module HydroPowerModels
 
 using JuMP, Clp, PowerModels, SDDP
-using Reexport
+import Reexport
+
+mutable struct HydroPowerModel
+    policygraph::SDDP.PolicyGraph
+    alldata::Array{Dict{Any,Any}}
+    params::Dict
+end
 
 include("variable.jl")
 include("constraint.jl")
 include("utilities.jl")
 include("IO.jl")
 include("simulate.jl")
 
-export  hydrothermaloperation, parse_folder, set_param, simulate_model,
+export  hydrothermaloperation, parse_folder, set_param,
         plotresults, plotscenarios, set_active_demand, flat_dict,
         descriptivestatistics_results, signif_dict
 
-@reexport using PowerModels, SDDP
+Reexport.@reexport using PowerModels, SDDP
 
 """
 data is a dict with all information of the problem. 
 
 param is a dict containing solution parameters.
 """
 function hydrothermaloperation(alldata::Array{Dict{Any,Any}}, params::Dict)
+    # verbose
+    if !params["verbose"]
+        PowerModels.silence()
+    end
 
     # Model Definition
-    m = SDDPModel(
+    policygraph = SDDP.LinearPolicyGraph(
                     sense   = :Min,
                     stages  = params["stages"],
-                    solver  = params["solver"],
-            objective_bound = 0.0
+                    optimizer  = with_optimizer(params["optimizer"]),
+                    lower_bound = 0.0,
+                    direct_mode=false
                                             ) do sp,t
         
+        # if set silence a solver
+        if !params["verbose"]
+            try
+                MOI.set(JuMP.backend(sp), MOI.Silent(), true)
+            catch
```

Comment:
Either we remove the catch or add som printing saying silent did not go through

Reply:
I think you should reference the issue from JuMP. Also, Guilherme fix issue, thus next JuMP version you should remove this.